### PR TITLE
[Lorpdate] Caligram is now Blacktide, adds new alt sprites for the Caligram-now-Blacktide clothing

### PR DESCRIPTION
**Pull request by Boviro**
## About The Pull Request
Edits the descriptions and names of the Caligram gear to be a generic Blacktide mercenary company, and adds alternate sprites to the Blacktide gear in a deep blue color (thank you for the sprites Caligram!)
NOTE: this PR has quite a story to it. The person who originally commissioned me to originally change the sprites has changed the scope of their project and i'm reusing this PR to fulfill some requests from the maintainers

## Why It's Good For The Game
Completes requests from the maintainer/admin team, also, the new sprites are very nice to look at.

## Proof Of Testing
~~No functional code was changed, purely flavor text. Also, it builds, so it's fine.~~
Doesn't do anything complicated, mostly flavor text changes and using the advanced reskin function. Tested locally, all sprites work, including digitigrade sprites. Builds.

## Changelog
:cl:
add: reflavors Caligram gear to Blacktide
add: adds alternate color sprites to the caligram gear
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
Wait a bit I am asking zydra and bombay about it

**Comment by Boviro** [comment]
Entirely fair, I'm not... 100% sure on all of this either and I would like the mod and coder staff to sign off on it before it gets merged.

**Comment by Boviro** [comment]
Also understandable. If it would be better to suffer some loadout bloat to add Blacktide variants of the Caligra stuff I can do that as well.

**Comment by projectkepler-ru** [comment]
Naw just wait abit, the caligras arent inactive we are just discussing the lore atm. I am waiting on the verdict. at latest it'll be friday but it shouldn't take that long

**Comment by Boviro** [comment]
Converted to draft via executive decision from the admin team to not go through with it. A wholly unique new set of sprites may be added for the Blacktide stuff, so I will leave this a draft until I get a concrete plan of where it will proceed.

**Comment by StrangeWeirdKitten** [comment]
I'll take over this PR

**Comment by Boviro** [comment]
Reflavors the caligram stuff to be a generic Blacktide, as opposed to the new proposal from harvey for the Molerats mercenary group, since I was asked to keep the change name. More importantly, it adds a new set of alt designs for the Caligram-now-Blacktide clothing, that being a deep blue color
image

File: modular_skyrat/master_files/code/modules/loadout/categories/donator_personal.dm [hunk 1]
```diff
@@ -133,32 +133,32 @@
 	//ckeywhitelist = list("enzoman")
 
 /datum/loadout_item/mask/nightlight_mask
-	name = "FIR-36 Rebreather"
+	name = "'Royez' Half-Face Rebreather"
 	item_path = /obj/item/clothing/mask/gas/nightlight
 //	ckeywhitelist = list("farsightednightlight", "raxraus", "1ceres", "marcoalbaredaa", "itzshift_yt", "drifter7371", "AvianAviator", "Katty Kat", "Investigator77", "Dalao Azure", "Socialistion", "ChillyLobster", "Sylvara", "AmZee", "Tf4", "rb303", "Kay_Nite", "whataboutism", "taac", "Halkyon", "Lupo_di_rosa", "Merek2", "lowpowermia", "RyeanBread", "Jesterz7", "Saund_Minah", "Ruediger4")
```
Comment (Majkl-J) [inline modular_skyrat/master_files/code/modules/loadout/categories/donator_personal.dm:138]: Feel free to remove the commented out whitelist it isn't coming back

**Review by Majkl-J — APPROVED** [review]
I have concluded that the comments can stay and whatnot

**Pull request merged**